main.cpp

```diff
@@ -109,15 +121,26 @@ struct Animal
     Animal();
     int numOfLegs;
     int numOfEyes;
+    int numOfToes;
     char animalClass;
 
     void classifyAnimal();
     bool isAnimalBipedal();
+
+    int toeCreator()
+    {
+        for(int i=0; i<numOfLegs; i++)
```

Comment:
add space around your operators
always prefer pre-increment/decrement over post-increment/decrement.
if you need an explanation, ask on Slack.
- [ ] revise